docs/src/app/app-routes.jsx

```diff
@@ -92,39 +92,54 @@ const AppRoutes = (
       <Route path="auto-complete" component={AutoCompletePage} />
       <Route path="avatar" component={AvatarPage} />
       <Route path="badge" component={BadgePage} />
+      <Redirect from="buttons" to="/components/buttons/flat-button" />
+      <Route path="buttons">
```

Comment:
Can we keep the url flat for the doc section?
I don't think that it's a good idea to nest it, as we don't nest our source folder.